task/backend/executor/task_executor.go

```diff
@@ -400,12 +400,10 @@ func (w *worker) executeQuery(p *promise) {
 
 	it.Release()
 
-	// log the statistics on the run
-	stats := it.Statistics()
-
-	b, err := json.Marshal(stats)
-	if err == nil {
-		w.te.tcs.AddRunLog(p.ctx, p.task.ID, p.run.ID, time.Now().UTC(), string(b))
+	// log the trace id and whether or not it was sampled into the run log
```

Comment:
@george there's a helper for this in the logger package that might make things a bit more readable

https://github.com/influxdata/influxdb/blob/master/logger/fields.go#L98

Reply:
@gavincabbage I decided to not use that as I already had the span off the context. What do you think. Split that helper into two parts?

Reply:
ah gotcha... yeah you could do that, the reflection makes things harder to grok, but I don't feel strongly either way